ledger-service/http-json/src/main/scala/com/digitalasset/http/HttpService.scala

```diff
@@ -62,8 +59,36 @@ object HttpService {
     def fromLedgerClientError(e: LedgerClientBase.Error): Error = Error(e.message)
   }
 
+  final case class LazyF[F[_], A, B](private val fn: A => F[B]) {
```

Comment:
given this is fairly generic , would make more sense for this to live in some`util` package

Reply:
I agree